forms-flow-api/src/formsflow_api/services/external/base_bpm.py

```diff
@@ -65,11 +65,19 @@ def post_request(
         return data
 
     @classmethod
-    def _get_headers_(cls, token, tenant_key=None, files=None):
+    def _get_headers_(cls, token, tenant_key=None, files=None, is_multitenant_client_secret=False):
         """Generate headers."""
         bpm_token_api = current_app.config.get("BPM_TOKEN_API")
-        bpm_client_id = current_app.config.get("BPM_CLIENT_ID")
-        bpm_client_secret = current_app.config.get("BPM_CLIENT_SECRET")
+        bpm_client_id = (
+            current_app.config.get("MULTITENANT_BPM_CLIENT_ID")
+            if is_multitenant_client_secret
+            else current_app.config.get("BPM_CLIENT_ID")
+        )
+        bpm_client_secret = (
+            current_app.config.get("MULTITENANT_BPM_CLIENT_SECRET")
+            if is_multitenant_client_secret
+            else current_app.config.get("BPM_CLIENT_SECRET")
+        )
```

Comment:
Can we change the `if is_multitenant_client_secret` conditions to `if tenant_key` ? `tenant_key` will only have value in a multi-tenant set up and is supposed to be `None` otherwise. We can also eliminate `is_multitenant_client_secret` as I don't see it used anywhere else.

Reply:
@salabh-aot , as discussed renamed the variable